packages/client/src/checkout/getCharges.ts

```diff
@@ -1,21 +0,0 @@
-import client, { adaptError } from '../helpers/client';
```

Comment:
Is the `/orders/{id}/charges` endpoint decommissioned?

Reply:
The mail goal of this task is fix this endpoint
The right path is:
`/orders/{id}/charges/{chargeId}`